dbscripts/xml/upgrade/3.1.0_update_a.xml

```diff
@@ -18,7 +18,7 @@
 		<query driver="postgres7">UPDATE submission_files SET genre_id = g.genre_id FROM genres g, submissions s WHERE submission_files.genre_id = 1 AND g.entry_key = 'SUBMISSION' AND g.context_id = s.context_id AND s.submission_id = submission_files.submission_id</query>
 	</sql>
 	<sql><!-- pkp/pkp-lib#2543 -->
-		<query>UPDATE filter_groups SET input_type = 'class::classes.article.Submission' WHERE symbolic = 'article=>doaj-json'</query>
+		<query>UPDATE filter_groups SET input_type = 'class::classes.submission.Submission' WHERE symbolic = 'article=>doaj-json'</query>
```

Comment:
@ajnyga, are there other persisted filters that need to be updated in the DB?

Reply:
ah of course, you are thinking of upgrading users. So those have to be updated with special queries?

class::classes.article.Article => class::classes.submission.Submission
class::classes.article.Article[] => class::classes.submission.Submission[]

class::classes.article.Author[] => class::classes.submission.Author[]

If we decide to rename ArticleGalley:
class::classes.article.ArticleGalley
class::classes.article.ArticleGalley[]

Reply:
Yes, that's right. I think there are some similar queries already in `dbscripts/xml/3.1.2_update.xml`.